packages/shared/components/popups/BackupStrongholdPopup.svelte

```diff
@@ -26,7 +26,7 @@
         </div>
     </div>
     <div class="flex flex-row justify-between space-x-4 w-full">
-        <Button secondary classes="w-1/2" onClick={handleCancelClick} disabled={busy}>
+        <Button outline classes="w-1/2" onClick={handleCancelClick} disabled={busy}>
```

Comment:
This button has also changed, before the background was visible, similar to this in figma 
![image](https://user-images.githubusercontent.com/3624944/190416527-34ed47cc-f5a1-4c01-9960-e87900167430.png)